mdsuite/project.py

```diff
@@ -225,7 +225,7 @@ def add_data(self, data_sets: dict):
         Updates the experiment classes.
         """
         for item in data_sets:
-            self.experiments[item].add_data(data_sets[item])
+            self.experiments[item].add_data(data_sets[item], file_format=file_format)
 
```

Comment:
This assumes that all files have the same file_format. Maybe also use a dictionary instead? We could use an `assert data_sets.keys() == file_format.keys()` to ensure that they are the same. Or we allow both, a single file format and a dictionary. Also docstrings.

Reply:
Should be solved in fa11735